schema/spacewalk/common/tables/rhnServerPackage.sql

```diff
@@ -38,5 +38,15 @@ CREATE UNIQUE INDEX rhn_sp_snep_uq
     
     ;
 
+CREATE INDEX rhn_server_package_nid_evrid_idx 
+    ON rhnServerPackage (name_id, evr_id)
+
+    ;
+
+CREATE INDEX rhn_sp_sien
+    ON rhnServerPackage USING btree (server_id, installtime, evr_id, name_id)
+
```

Comment:
These empty lines are not needed. also the semicolon on an extra line is not required.
All this were leftovers from the time we used `sed` to change oracle style indexes to postgres indexes.